source/iopipe/json/serialize.d

```diff
@@ -979,14 +840,14 @@ unittest
     static class C
     {
         int x;
-        static C fromJSON(JT)(ref JT tokenizer, ReleasePolicy relPol)
+        static C fromJSON(P, JT)(ref JT tokenizer, ref P policy)
```

Comment:
Need to test both relpol and policy options, so add a second class with the policy design.